lib/src/test/resources/generators/play-26-apidoc-api.txt

```diff
@@ -730,38 +730,12 @@ package io.apibuilder.api.v0.models {
     import io.apibuilder.common.v0.models.json._
     import io.apibuilder.generator.v0.models.json._
     import io.apibuilder.spec.v0.models.json._
-
-    private[v0] implicit val jsonReadsUUID = __.read[String].map(java.util.UUID.fromString)
-
-    private[v0] implicit val jsonWritesUUID = new Writes[java.util.UUID] {
-      def writes(x: java.util.UUID) = JsString(x.toString)
-    }
-
-    private[v0] implicit val jsonReadsJodaDateTime = __.read[String].map { str =>
-      import org.joda.time.format.ISODateTimeFormat.dateTimeParser
-      dateTimeParser.parseDateTime(str)
-    }
-
-    private[v0] implicit val jsonWritesJodaDateTime = new Writes[org.joda.time.DateTime] {
-      def writes(x: org.joda.time.DateTime) = {
-        import org.joda.time.format.ISODateTimeFormat.dateTime
-        val str = dateTime.print(x)
-        JsString(str)
-      }
-    }
-
-    private[v0] implicit val jsonReadsJodaLocalDate = __.read[String].map { str =>
-      import org.joda.time.format.ISODateTimeFormat.dateParser
-      dateParser.parseLocalDate(str)
-    }
-
-    private[v0] implicit val jsonWritesJodaLocalDate = new Writes[org.joda.time.LocalDate] {
-      def writes(x: org.joda.time.LocalDate) = {
-        import org.joda.time.format.ISODateTimeFormat.date
-        val str = date.print(x)
-        JsString(str)
-      }
-    }
+    import play.api.libs.json.Writes._
+    import play.api.libs.json.Reads._
+    import play.api.libs.json.JodaReads._
+    import play.api.libs.json.JodaReads.DefaultJodaLocalDateReads
```

Comment:
Isn't `DefaultJodaLocalDateReads` imported with `play.api.libs.json.JodaReads._` ?

Reply:
Strangely it isn't...! Couldn't quite figure it out why, but it's an `implicit object`